### ISSUE TITLE
System throws an error when there are blank lines at the end of the imported CSV file

### ISSUE THREAD

**Issue by yostashiro** [rfhk/she-custom#4]
## Steps to reproduce

Import a CSV file with blank lines (lines only with commas) through `Sales Order Import` wizard.

![_102615_113912_am](https://cloud.githubusercontent.com/assets/7766939/10720823/67ae43f4-7bd6-11e5-835f-88042241d49c.jpg)
## Current behavior

Following error is thrown.

```
Odoo Server Error
Traceback (most recent call last):
  File "/home/roomsfor/ws/odoo/8.0/openerp/http.py", line 537, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/roomsfor/ws/odoo/8.0/openerp/http.py", line 574, in dispatch
    result = self._call_function(**self.params)
  File "/home/roomsfor/ws/odoo/8.0/openerp/http.py", line 310, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/roomsfor/ws/odoo/8.0/openerp/service/model.py", line 113, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/roomsfor/ws/odoo/8.0/openerp/http.py", line 307, in checked_call
    return self.endpoint(*a, **kw)
  File "/home/roomsfor/ws/odoo/8.0/openerp/http.py", line 803, in __call__
    return self.method(*args, **kw)
  File "/home/roomsfor/ws/odoo/8.0/openerp/http.py", line 403, in response_wrap
    response = f(*args, **kw)
  File "/home/roomsfor/ws/odoo/8.0/addons/web/controllers/main.py", line 948, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/roomsfor/ws/odoo/8.0/addons/web/controllers/main.py", line 936, in _call_kw
    return getattr(request.registry.get(model), method)(request.cr, request.uid, *args, **kwargs)
  File "/home/roomsfor/ws/odoo/8.0/openerp/api.py", line 241, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/roomsfor/ws/odoo/8.0/openerp/api.py", line 363, in old_api
    result = method(recs, *args, **kwargs)
  File "/home/roomsfor/ws/odoo/she-custom/sale_order_import/wizard/import_sale.py", line 271, in import_sale_data
    qty = float(row[product_qty].strip())
ValueError: could not convert string to float: 
```
## Expected behavior

No error should show.  The program should ignore lines with no content.

**Comment by yostashiro** [comment]
@mra-odoo

**Comment by yostashiro** [comment]
Fixed with this commit: 9b157a3a5cd58cf3a1ba49dc81a730b0b2fdd40f
